src/pudl/output/eia.py

```diff
@@ -114,7 +114,7 @@ def out_eia__yearly_plants(
     },
     compute_kind="Python",
 )
-def out_eia__yearly_generators(
+def _out_eia__yearly_generators(
```

Comment:
I made this an intermediate table because the old `mcoe_generators_{freq}` table (now `out_eia__{freq}_generators`) has all of the same attributes plus the valuable derived attributes.